pegasus/emails/volunteer_contact_2015_receipt.md

```diff
@@ -0,0 +1,26 @@
+---
+from: '"Tanya Parker (Code.org)" <tanya_parker@code.org>'
+to: <%= volunteer_name_s + " <" + volunteer_email_s + ">" %>
+subject: "A teacher is requesting your help for the Hour of Code"
+---
+
+<% remove_link = "http://#{CDO.canonical_hostname('code.org')}/v2/forms/VolunteerEngineerSubmission2015/#{form.secret}/delete" %>
```

Comment:
I suspect that this won't work because this route is expecting a POST not a GET. The way this works for canceling workshops is that the users are sent a link to a regular page that has a button to do the actual canceling (and text about what they are doing and "Are you sure", etc), maybe you could do it that way. That would allow you to show the user a nice message handle cases like the user going to the page twice (so you can tell them that they are already deleted the second time).